backend/test/integration/langchain.test.ts

```diff
@@ -49,16 +51,15 @@ jest.mock('langchain/embeddings/openai', () => {
 	};
 });
 
-class MockMemoryVectorStore {
-	asRetriever() {
-		mockAsRetriever();
-	}
-}
```

Comment:
@pmarsh-scottlogic It seemed unnecessary to create a class, so I inlined it below.